rdt/transformers/base.py

```diff
@@ -367,8 +367,8 @@ def _fit(self, columns_data):
 
     def _set_seed(self, data):
         hash_value = self.columns[0]
-        for value in data.head(5):
-            hash_value += str(value)
+        for _, row in data.head(5).iterrows():
+            hash_value += str(row[self.columns[0]])
```

Comment:
do we have a simple unit test that checks that if only one value is different for two columns the seed is different?